client/src/app/site/pages/meetings/pages/autopilot/components/poll-collection/poll-collection.component.ts

```diff
@@ -109,8 +109,12 @@ export class PollCollectionComponent<C extends PollContentObject> extends BaseCo
         const model = contentObject.getVerboseName();
         const pollTitle = poll.getTitle();
 
-        if (this.showExtendedTitle && contentObject?.fqid !== this.currentProjection?.fqid) {
-            return `(${model}) ${listTitle} - ${pollTitle}`;
+        if (
+            this.showExtendedTitle &&
+            contentObject?.fqid !== this.currentProjection?.fqid &&
+            [`Topic`, `Election`].includes(model)
```

Comment:
The verbose names should not be used for checks. You could instead use `contentObject.COLLECTION` and then check against `Topic.COLLECTION` and `Assignment.COLLECTION`.